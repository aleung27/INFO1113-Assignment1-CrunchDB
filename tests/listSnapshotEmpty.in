LIST SNAPSHOTS
SNAPSHOT
SNAPSHOT
SNAPSHOT
LIST SNAPSHOTS
CHECKOUT 1
LIST SNAPSHOTS
ROLLBACK 1
LIST SNAPSHOTS
DROP 1
LIST SNAPSHOTS
