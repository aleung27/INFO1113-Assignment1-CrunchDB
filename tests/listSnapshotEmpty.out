> no snapshots

> saved as snapshot 1

> saved as snapshot 2

> saved as snapshot 3

> 3
2
1

> ok

> 3
2
1

> ok

> 1

> ok

> no snapshots

